backend/app/views/backoffice/base/_account_fields.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= f.simple_fields_for :account do |ff|  %>
```

Comment:
no sure if I am convinced to have this at `base` folder when this is gonna be valid just for PD and Investor. What about Account folder?